node/src/runtime/mainnet.rs

```diff
@@ -23,22 +23,6 @@ impl ValidatorIdentity {
 			},
 		}
 	}
-
-	pub fn from_pub(ed: &str, sr: &str, _ecdsa: &str) -> ValidatorIdentity {
```

Comment:
Let's keep it for future. it's easier to manipulate using this. At least for future.
#[allow(dead_code)]

Reply:
Done.